src/log.cpp

```diff
@@ -99,4 +143,10 @@ void Log::add(const QString &text)
     // Add it to the open log window, if any
     if ( !mDialog.isNull() )
         mDialog.data()->add( logline );
+
```

Comment:
```suggestion
        QTextStream(stderr) << logline << endl;
```
I think adding this here would make sense. We always want to log to `stderr`, just as we always want to log to the log viewer.

Reply:
I'm not sure about how this aligns with an initial purpose of recently removed `--debug flag, but I'd rather have a somewhat meaningful log in stderr than have none, so I like this idea.